src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Layers/ArcGISTiledLayer/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Name": "ArcGIS Tiled Layer",
+  "SampleName": "ArcGISTiledLayer",
+  "Description": "The Tiled Layer from URL app is one of the most basic Map apps in the SDK for .NET using an ArcGISTiledLayer basemap from an ArcGIS Online service URL. It shows how to add a MapView, Map and  ArcGISTiledLayer to the apps XAML from an ArcGIS Online service URL and bind that to a Basemap. The Basemap is used to create a Map which is used inside of the MapView. By default, this map supports basic zooming and panning operations.",
```

Comment:
The description is very misleading. We don't have 'apps' in the dotnet samples, we just have samples. Android uses separate apps, but we have everything included in a single application and the samples are exposed as a usercontrols. We should talk only about samples ie. the Tiled Layer form URL sample is one of the most basic examples how to use ArcGISTiledLayer...